spec/session_tracker_spec.rb

```diff
@@ -3,13 +3,17 @@
 require 'spec_helper'
 require 'json'
 
+module Bugsnag
+  class SessionTracker
+    attr_accessor :track_sessions
```

Comment:
What's this for?

Reply:
It was being used to access a variable to stop the delivery threads being spun up, but is obsolete now.